test/built-ins/Proxy/enumerate/call-parameters.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.11
+description: >
+    [[Enumerate]] ()
+
+    8. Let trapResult be Call(trap, handler, «target»).
+---*/
+
+var x, _target, _handler;
+var target = {
+    attr: 1
+};
+var handler = {
+    enumerate: function(t) {
+        _target = t;
+        _handler = this;
+        return {next: function() { return { done:true }; } };
```

Comment:
I would insist in this test for the same reason it catches an ES6 issue on a specific browser.

While this can't be addressed would it be possible to blacklist this test there?